docs/_newsfragments/1589.breakingchange.rst

```diff
@@ -0,0 +1,3 @@
+Deserializing an empty JSON body on a request will return a BadRequest.
```

Comment:
will return a BadRequest :arrow_right: will raise an instance of `HTTPBadRequest`.